package.json

```diff
@@ -19,13 +19,14 @@
   "license": "Apache-2.0",
   "dependencies": {
     "archiver": "^2.1.1",
-    "caporal": "^0.9.0",
```

Comment:
Why update this version? I'm not against it, just curious what prompted the upgrade. I couldn't find any release notes for their versions.

Reply:
NPM updated those when I added moment, didn't seem to cause any problems so I left them.

Reply:
Ah, you should be using `yarn` when adding dependencies. That's why `yarn.lock` wasn't updated. Please undo the `package.json` updates locally then re-add moment using yarn.